jamiehs/bootstrap-sass-middleman
# Bethany/Upgrade to Middleman 4, Remove Livereload

Removed Compass, added Autoprefixer instead. 
Removed LiveReload, use Browsersync with Node instead
### To Test
- Run `bundle install` to install updated Gems
- Start `middleman`
- Start `browser-sync start --proxy 'localhost:4567' --files source` in separate terminal file and open browser, navigate to localhost:4567 edit a CSS file and index.html file to confirm that LiveReload is working
- Run `middleman build` to make sure there are no errors
#### Possible bugs/downsides

Can not get bs-config file to work so can not pass options to Browsersync outside of command line
